_src/pattern-library/sass/base/_reset.scss

```diff
@@ -104,6 +104,13 @@ pre {
 // ----------------------------
 .sr-only {
     @extend %a11y-hide;
+    background: black !important; // providing contrast
```

Comment:
I chose to use `!important` here to block any overrides that occur later. These elements aren't seen anyway, but we need to provide sufficient contrast for automated checkers.